test/functional/install/install-bad.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -n test-bundle -f /file_1,/file_2 "$TEST_NAME"
+	sudo mkdir -p "$TARGETDIR"/usr/share/clear/bundles
```

Comment:
this line doesn't hurt, but just FYI, this directory is already created with the test environment:
https://github.com/clearlinux/swupd-client/blob/master/test/functional/testlib.bash#L1326